arista/utils/sonic_sfputil.py

```diff
@@ -49,6 +48,14 @@ def __init__(self):
 
     class SfpUtilNative(SfpUtilCommon):
         """Native Sonic SfpUtil class"""
+        XCVR_PRESENCE_POLL_PERIOD_SECS = 1
+
+        def __init__(self):
```

Comment:
Missing call to parent `__init__` method.
However it's currently not really a problem since it's empty
https://github.com/Azure/sonic-platform-common/blob/ee122155e0f6e40b483d24737c5f1b729bdbf111/sonic_platform_base/sonic_sfp/sfputilbase.py#L220